### ISSUE TITLE
`drush dl valkyrie` says there are no stable releases

### ISSUE THREAD

**Issue by MatthewHager** [GetValkyrie/valkyrie#47]

**Comment by RealLukeMartin** [comment]
I got this when helping Justin set up valkyrie today on his machine. After hitting that error, I had him do a `brew reinstall drush --HEAD` and it fixed the issue for him. Could be another underlying issue causing this, but worth a try.

**Comment by MatthewHager** [comment]
I don't think that the `brew reinstall drush --HEAD` was a real fix. I've now confirmed that drush 6.5.0 will download valkyrie just fine. The latest head version of drush will not see the release every time.

```
➜  omega 2  drush --debug dl valkyrie
Starting Drush preflight. [0.01 sec, 2.17 MB]                                                                                                                          [preflight]
Cache HIT cid: 7.0-dev-commandfiles-0-1d9944fd8d92d646f8e3bf3c931e0b08 [0.01 sec, 2.23 MB]                                                                                 [debug]
Bootstrap to phase 0. [0.08 sec, 6.35 MB]                                                                                                                              [bootstrap]
Found command: pm-download (commandfile=pm) [0.09 sec, 6.35 MB]                                                                                                        [bootstrap]
Loading version_control engine. [0.09 sec, 6.36 MB]                                                                                                                       [notice]
Loading package_handler engine. [0.09 sec, 6.37 MB]                                                                                                                       [notice]
Executing: wget --version
Loading release_info engine. [0.09 sec, 6.4 MB]                                                                                                                           [notice]
Including /usr/local/Cellar/drush/HEAD/libexec/commands/pm/download.pm.inc [0.1 sec, 6.44 MB]                                                                          [bootstrap]
Calling hook drush_pm_download_validate [0.1 sec, 6.52 MB]                                                                                                                 [debug]
Returned from hook drush_pm_download_validate [0.1 sec, 6.52 MB]                                                                                                           [debug]
Calling hook drush_pm_download [0.1 sec, 6.54 MB]                                                                                                                          [debug]
Downloading release history from https://updates.drupal.org/release-history/valkyrie/7.x [0.1 sec, 6.65 MB]                                                               [notice]
/Users/matthew/.drush/cache/download/https---updates.drupal.org-release-history-valkyrie-7.x retrieved from cache. [0.1 sec, 6.65 MB]                                     [notice]
There are no stable releases for project valkyrie. [0.1 sec, 6.66 MB]                                                                                                  [warning]
Choose one of the available releases for valkyrie:
 [0]  :  Cancel
```

**Comment by ergonlogic** [comment]
This was a bug in Drush. I fixed it in PR drush-ops/drush #1296
